install_config/topics/configuring_rhv_objects.adoc

```diff
@@ -1,66 +1,425 @@
++
+[options="nowrap" subs="+quotes,verbatim"]
+----
+# ansible-vault create secure_vars.yaml
+----
+
+. Add the following variables to *_secure_vars.yaml_*:
++
+[source,yml]
+----
+engine_password: <Manager_password>
+bastion_root_password: <bastion_root_password>
+rhsub_user: <Red_Hat_Subscription_Manager_username>
+rhsub_pass: <Red_Hat_Subscription_Manager_password>
+rhsub_pool: <Red_Hat_Subscription_Manager_pool_id>
+root_password: <OpenShift_node_root_password>
+engine_cafile: <Engine_CA_certificate>
+oreg_auth_user: <image_registry_authentication_username>
+oreg_auth_password: <image_registry_authentication_password>
+----
+
+. Save the file and record the vault password.
+
+. Create the installation playbook, *_/bastion_installation/create-bastion-machine-playbook.yaml_*, and update its parameters:
++
+[source,yml]
+----
+---
+- name: Create a bastion machine
+  hosts: localhost
+  connection: local
+  gather_facts: false
+
+  vars:
+    engine_url: https://_Manager_FQDN_/ovirt-engine/api
+    engine_user: <admin@internal>
+    engine_password: "{{ engine_password }}"
+    engine_cafile: /etc/pki/ovirt-engine/ca.pem
+
+    qcow_url: <1>
+    template_cluster: Default
+    template_name: rhelguest7
+    template_memory: 4GiB
+    template_cpu: 2
+    wait_for_ip: true
+    debug_vm_create: false
 
-To integrate {product-title} with Red Hat Virtualization, take the following
-actions as part of your
-xref:../install/host_preparation.adoc#install-config-install-host-preparation[host preparation].
+    vms:
+    - name: rhel-bastion
+      cluster: "{{ template_cluster }}"
+      profile:
+        cores: 2
+        template: "{{ template_name }}"
+        root_password: "{{ root_password }}"
+        ssh_key: "{{ lookup('file', '/root/.ssh/id_rsa_ssh_ocp_admin.pub') }}"
+        state: running
+      cloud_init:
+        custom_script: |
+          rh_subscription:
+            username: "{{ rhsub_user }}"
+            password: "{{ rhsub_pass }}"
+            auto-attach: true
+            disable-repo: ['*']
+            # 'rhel-7-server-rhv-4.2-manager-rpms' supports RHV 4.2 and 4.3
+            enable-repo: ['rhel-7-server-rpms', 'rhel-7-server-extras-rpms', 'rhel-7-server-ansible-2.7-rpms', 'rhel-7-server-ose-3.11-rpms', 'rhel-7-server-supplementary-rpms', 'rhel-7-server-rhv-4.2-manager-rpms']
+          packages:
+            - ansible
+            - ovirt-ansible-roles
+            - openshift-ansible
+            - python-ovirt-engine-sdk4
+  pre_tasks:
+    - name: Create an ssh key-pair for the OCP admin
+      user:
+        name: root
+        generate_ssh_key: yes
+        ssh_key_file: .ssh/id_rsa_ssh_ocp_admin
 
-. To provide high availability in case of the loss of one hypervisor host,
-add each class of instance to a negative affinity group. See
-link:https://ovirt.org/develop/release-management/features/sla/vm-affinity/[VM Affinity].
+  roles:
+    - oVirt.image-template
+    - oVirt.vm-infra
+  no_log: true
```

Comment:
@rgolangh This should probably go to the top level of the play, i.e. to line 51 or something like this

Reply:
Done. Please review.

Reply:
What will the effect of this be on all the log? Just verify that we are left with log you can work with in case of failures.